### ISSUE TITLE
Error starting server

### ISSUE THREAD

**Issue by paglias** [HabitRPG/habitica#3961]
I get:

```
##############################################################
#
#   !!! MONGOOSE WARNING !!!
#
#   This is an UNSTABLE release of Mongoose.
#   Unstable releases are available for preview/testing only.
#   DO NOT run this in production.
#
##############################################################


/home/matteo/Development/habitrpg/node_modules/mongoose/lib/schema.js:360
    throw new TypeError('Undefined type at `' + path +
          ^
TypeError: Undefined type at `paths.model`
  Did you try nesting Schemas? You can only nest using refs or arrays.
    at Function.Schema.interpretAsType (/home/matteo/Development/habitrpg/node_modules/mongoose/lib/schema.js:360:11)
    at Schema.path (/home/matteo/Development/habitrpg/node_modules/mongoose/lib/schema.js:303:29)
    at Schema.add (/home/matteo/Development/habitrpg/node_modules/mongoose/lib/schema.js:217:12)
    at Schema.add (/home/matteo/Development/habitrpg/node_modules/mongoose/lib/schema.js:212:14)
    at new Schema (/home/matteo/Development/habitrpg/node_modules/mongoose/lib/schema.js:73:10)
    at Mongoose.model (/home/matteo/Development/habitrpg/node_modules/mongoose/lib/index.js:286:14)
    at module.exports.init (/home/matteo/Development/habitrpg/node_modules/mongoose-id-autoinc/lib/mongoose-id-autoinc.js:103:18)
    at Object.<anonymous> (/home/matteo/Development/habitrpg/src/server.js:49:11)
    at Module._compile (module.js:456:26)
    at Object.Module._extensions..js (module.js:474:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:312:12)
    at Function.Module.runMain (module.js:497:10)
    at startup (node.js:119:16)
    at node.js:906:3
```

**Comment by lefnire** [comment]
which mongoose version you on after `npm install`? I reverted the mongoose upgrade, is the older version unstable? See https://github.com/LearnBoost/mongoose/issues/1920 for the reason I upgraded then downgraded

**Comment by paglias** [comment]
@lefnire 3.8.5 I don't know why it continues to say it's unstable 
![selezione_002](https://cloud.githubusercontent.com/assets/589911/4110307/65cb5a6c-31f0-11e4-81fa-1aa3af249f12.jpg)

**Comment by paglias** [comment]
@lefnire I'm with an empty database except for having run ./src/seed.js because I deleted the old one (it had a name conflict with the snapshot of the prod db)

**Comment by lefnire** [comment]
@paglias I ran `rm -rf npm_modules && npm install`, can't replicate. I'll try to replicate again when I get the chance, ideally https://github.com/LearnBoost/mongoose/issues/1920 will get fixed soon so we can just upgrade to mongoose#latest (which will likely stop that error)

**Comment by paglias** [comment]
@lefnire cloned again and it's working

**Comment by betaveros** [comment]
In case anybody else runs into this: I also got this error (with a new vagrant instance on the latest commit of `develop`), and downgrading mongoose didn't change anything for me (I tried both `npm install mongoose@3.8.5` and `npm install mongoose@3.8.17`), but running `rm -rf node_modules && npm install` worked. It takes a while, of course.

edit: The !!! MONGOOSE WARNING !!! still appears, but it doesn't seem to be a problem.
